alerter.js

```diff
@@ -1,26 +1,24 @@
-let alertFailureCount = 0;
+const { expect } = require('chai');
 
-function networkAlertStub(celcius) {
-    console.log(`Alert! Temperature is ${celcius} degrees`);
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
-}
+const { networkAlertStub } = require('./alertStub');
+
+let alertFailureCount = 0;
 
 function alertInCelcius(farenheit) {
-    const celcius = (farenheit - 32) * 5 / 9;
-    const returnCode = networkAlertStub(celcius);
-    if (returnCode != 200) {
-        // non-ok response is not an error! Issues happen in life!
-        // let us keep a count of failures to report
-        // However, this code doesn't count failures!
-        // Add a test below to catch this bug. Alter the stub above, if needed.
-        alertFailureCount += 0;
-    }
+	const celcius = ((farenheit - 32) * 5) / 9;
+	const returnCode = networkAlertStub(celcius);
+	if (returnCode != 200) {
+		// non-ok response is not an error! Issues happen in life!
+		// let us keep a count of failures to report
+		// However, this code doesn't count failures!
+		// Add a test below to catch this bug. Alter the stub above, if needed.
+		console.log(returnCode);
+		alertFailureCount += 1;
+	}
 }
 
 alertInCelcius(400.5);
 alertInCelcius(303.6);
 console.log(`${alertFailureCount} alerts failed.`);
+expect(alertFailureCount).equals(0);

```

Comment:
The functionality is to have one failure (at least). Reading this test back as the expected functionality: "have no failures". There's more to an alerter, isn't there?